oak_functions/examples/key_value_lookup/config.toml

```diff
@@ -1,6 +1,7 @@
 lookup_data_url = "https://storage.googleapis.com/oak_lookup_data/lookup_data_weather"
 lookup_data_download_period = "10m"
 worker_threads = 4
+tee_certificate = "./examples/certs/local/ca.pem"
```

Comment:
I don't think this certificate is the TEE certificate, is it? AFAICT it's just the root cert that we use for testing TLS connections.

Reply:
I have copied this parameter from `experimental/grpc_attestation/src/main.rs`. 
`tee_certificate` is needed for creating an instance of `AttestationServer`, but I don't quite understand what is does. @ipetr0v Could you please clarify what it is for?

Reply:
It's just currently used as a test certificate for TEE purposes. When we will have remote attestation, we will be able to use authentic certificates instead of this.